packages/esm-patient-orders-app/src/index.ts

```diff
@@ -45,3 +46,5 @@ export const ordersDashboardLink =
   );
 
 export const ordersDashboard = getSyncLifecycle(OrdersSummary, options);
+
+export const cancelOrderForm = getSyncLifecycle(CancelOrderForm, options);
```

Comment:
This should use `getAsyncLifecycle` for workspaces.

Reply:
General principle is that `getAsyncLifecycle()` should be preferred unless `getSyncLifecycle()` is absolutely necessary (this basically just means "is not a link rendered in the left nav or breadcrumb", etc.). Please help us keep O3 performant!

Reply:
https://o3-docs.openmrs.org/docs/migrate-to-core-v5#6-use-fewer-dynamic-imports-for-improved-performance